x-pack/plugins/lists/server/routes/find_list_item_route.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter } from 'kibana/server';
+
+import { LIST_ITEM_URL } from '../../common/constants';
+import {
+  buildRouteValidation,
+  buildSiemResponse,
+  transformError,
+  validate,
+} from '../siem_server_deps';
+import { findListItemSchema, foundListItemSchema } from '../../common/schemas';
+import { decodeCursor } from '../services/utils';
+
+import { getListClient } from './utils';
+
+export const findListItemRoute = (router: IRouter): void => {
+  router.get(
+    {
+      options: {
+        tags: ['access:lists'],
+      },
+      path: `${LIST_ITEM_URL}/_find`,
+      validate: {
+        query: buildRouteValidation(findListItemSchema),
+      },
+    },
+    async (context, request, response) => {
+      const siemResponse = buildSiemResponse(response);
+      try {
+        const lists = getListClient(context);
+        const {
+          cursor,
+          filter: filterOrUndefined,
+          list_id: listId,
+          page: pageOrUndefined,
+          per_page: perPageOrUndefined,
+          sort_field: sortField,
+          sort_order: sortOrder,
+        } = request.query;
+
+        const page = pageOrUndefined ?? 1;
```

Comment:
It's ok here, but for most other things I still like at the schema level of defaults so when I reuse the atoms of the schema in multiple spots I get the defaults and if I need to change the defaults they are in one spot.

However, this might show how I like to push defaults as high up as I possibly towards the top of the REST requests. That's always the best approach IMHO.